lib/gnat/jetstream/pull_consumer/server.ex

```diff
@@ -59,26 +63,38 @@ defmodule Gnat.Jetstream.PullConsumer.Server do
         } = gen_state
       ) do
     Logger.debug(
-      "#{__MODULE__} for #{stream_name}.#{consumer_name} is connecting to Gnat.",
+      "#{__MODULE__} for #{stream_name}.#{gen_state.consumer_name} is connecting to Gnat.",
       module: module,
       listening_topic: listening_topic,
       connection_name: connection_name
     )
 
     with {:ok, conn} <- connection_pid(connection_name),
-         Process.link(conn),
```

Comment:
This is a bug IMO, we shouldn't be killing the connection if a PullConsumer crashes, so I've changed this to a monitor so we can reconnect when the connection crashes/restarts, but the connection doesn't care if we crash.

Reply:
Yes, clients should try to resume consumers, never crash the connection.